project/cmake/addons/addons/pvr.mediaportal.tvserver/pvr.mediaportal.tvserver.txt

```diff
@@ -1 +1 @@
-pvr.mediaportal.tvserver https://github.com/kodi-pvr/pvr.mediaportal.tvserver 87422e6
\ No newline at end of file
+pvr.mediaportal.tvserver https://github.com/kodi-pvr/pvr.mediaportal.tvserver 606a070
```

Comment:
Also here to 51e68c4.

Related to https://github.com/kodi-pvr/pvr.mediaportal.tvserver/pull/2.